app/src/main/java/com/breadwallet/tools/security/AuthManager.java

```diff
@@ -214,7 +214,7 @@ public void authPrompt(final Context context, String title, String message, bool
                 transaction.setCustomAnimations(0, 0, 0, R.animator.plain_300);
                 transaction.add(android.R.id.content, fingerprintFragment, FragmentFingerprint.class.getName());
                 transaction.addToBackStack(null);
-                if (!app.isDestroyed())
+                if (!app.isDestroyed() && !app.isFinishing())
```

Comment:
assuming this will block action when the object is still alive.

Reply:
exactly, since the next line was crashing the app in some cases, it's better to be extra careful. if the activity or fragment is starting to finish, we shouldn't attempt to navigate.